conf.py

```diff
@@ -13,21 +13,34 @@
 copyright = "%s, %s" % (datetime.date.today().year, author)
 release = '1.0'
 
+html_context = {
+    # Change to the discourse instance you want to be able to link to
+    "discourse_prefix": "https://discourse.ubuntu.com/t/",
+    # Change to the GitHub info for your project
+    "github_url": "https://github.com/canonical/starter-pack",
```

Comment:
In readme.rst, perhaps add html_context to the conf.py settings that need checking/editing. These defaults may be missed otherwise.

Reply:
Oh right - I was planning to update those instructions, but thought they were in the different repo. But they are in the readme here, thanks for pointing that out!